### PR TITLE
Added docs on how to access neurons created with quill using the NNS …

### PR DESCRIPTION
**Pull request by csepreghy**
…dapp and Ledger

Thank you for your contribution to the IC Developer Portal. This repo contains the content for https://internetcomputer.org and the ICP Developer Documentation, https://internetcomputer.org/docs/. 

If you are submitting a Pull Request for adding or changing content on the ICP Developer Documentation, please make sure that your contribution meets the following requirements:

- [ ] Follows the [developer docs style guide](style-guide.md).
- [ ] Follows the [best practices and guidelines](README.md#best-practices).
- [ ] New documentation pages include [document tags](README.md#document-tags).
- [ ] New documentation pages include [SEO keywords](README#seo-keywords).
- [ ] New documents are in the `.mdx` file format to support the previous two components. 
- [ ] New documents are registered in [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js), otherwise, it will not appear in the
  side navigation bar.
- [ ] Make sure that the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file is
  filled with new documents that you added. This way we can ensure that future Pull Requests are reviewed by the right people.

### CODE REVIEW

File: docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-manage-quill-neurons.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Overview
+You may have neurons controlled by a principal ID that was created using [quill](../../../../../../docs/current/developer-docs/developer-tools/cli-tools/quill-cli-reference/quill-parent). In this case, it may be inconvenient to manage your neurons through the terminal. If you know the seed phrase of this principal ID, you can use a Ledger hardware wallet to manage your neurons through the [NNS dapp](/https://nns.ic0.app/).
+
+## How to access neurons controlled by a principal ID created with quill
+
+- **Step 1**: Unlock your Ledger device and navigate to **Settings Manage device** > **Security** > **Reset device**
+
+![NNS Dapp Ledger 1](../../_attachments/ledger-1.webp)
+
+- **Step 2**: Type in your PIN to reset your Ledger device.
```
Comment (LaraAS) [inline docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-manage-quill-neurons.mdx:23]: Why do you need to reset? Should this maybe come with a big warning that you should only do this if you don't user this ledger for anything else?

File: docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-manage-quill-neurons.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- **Step 3**: After your Ledger device has been successfully reset, navigate to **Restore from Recovery phase**.
```
Comment (LaraAS) [inline docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-manage-quill-neurons.mdx:24]:
```suggestion
- **Step 3**: After your Ledger device has been successfully reset, navigate to **Restore from Recovery phrase**.
```

File: docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-manage-quill-neurons.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+![NNS Dapp Ledger 7](../../_attachments/ledger-7.webp)
+
+- **Step 10**: Select the ICP account you just created
+
+- **Step 11**: Click **Show Neurons** and approve the transaction on your Ledger device
+
+![NNS Dapp Ledger 8](../../_attachments/ledger-8.webp)
+
+- **Step 12**: Clicking Add to NNS Dapp will add your NNS Dapp principal ID as a hotkey to the neurons. This allows you to see and manage your neurons using the NNS Dapp. For all actions outside of voting and changing following, you will need to use your Ledger device to approve transactions
```
Comment (LaraAS) [inline docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-manage-quill-neurons.mdx:56]:
```suggestion
- **Step 12**: Click Add to NNS Dapp to add your NNS Dapp principal ID as a hotkey to the neurons. This allows you to see and manage your neurons using the NNS Dapp. For all actions outside of voting and changing following, you will need to have your Ledger device connected and approve the action.
```

(would also write as "do this")

File: docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-manage-quill-neurons.mdx (hunk 1 above)
Comment (csepreghy) [reply docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-manage-quill-neurons.mdx:23]: You need the Ledger device to use the same seed phrase that your principal ID has that was created with quill. As far as I understand your Ledger can only have one private key. Great point about the warning though!

**Review by reigj1 — APPROVED** [review]

**Pull request merged**